dhis-2/dhis-support/dhis-support-test/src/main/java/org/hisp/dhis/web/WebClient.java

```diff
@@ -152,13 +152,10 @@ default HttpResponse PATCH(String url, String body) {
         HttpMethod.PATCH, url, ContentType("application/json-patch+json"), Body(body));
   }
 
-  default HttpResponse PATCH_OLD(String url, String body) {
```

Comment:
The naming `PATCH_OLD` maybe was a bit misleading. Because this is a general client so this isn't in any way connected to the CRUD old vs. new patch. We may have endpoints that do a `PATCH` HTTP method and this client should have a way to call them. So in hindsight we should have name this `PATCH_JSON` or something to make clear that this is sending a patch with a JSON body if there is one.